app/views/plans/_share_form.html.erb

```diff
@@ -30,7 +30,10 @@
       </tr>
     </thead>
     <tbody>
-      <% plan_roles = @plan.roles.where(active: true) %>
+      <% 
+        # Exclude reviewers from the list 
+        plan_roles = @plan.roles.where("active = ? AND access < ?", true, 16) 
```

Comment:
the bit field gem we are using got nice build in helpers to avoid hardcoding the access integer. For example, @plan.roles.where({ active: true}).where(Role.not_reviewer_condition)

Reply:
ah ok. I will check out the gem then. I tried :reviewer but it was not working. will look into that

Reply:
refactored and move line to controller as well